development-tools/clinic.rst

```diff
@@ -1495,6 +1495,26 @@ You can still use a self converter, a return converter, and specify
 a *type* argument to the object converter for :c:macro:`METH_O`.
 
 
+How to convert var-positional parameter functions
+-------------------------------------------------
+
+To convert a var-positional parameter function,
+prepending the parameter name with ``*`` ,
+and the parameter should use the ``object`` converter::
+
+   /*[clinic input]
+   var_positional_sample
+
+       foo: int
+       *args: object
```

Comment:
Is this true? In `libclinic` I only see a `tuple` and `array` converter as subclasses of VarPosCConverter, and I can't see any uses of `*arg_name: object` in the tree, though there are plenty of uses of `*args : tuple` or `*args: array`.